assignment-client/src/scripts/EntityScriptServer.cpp

```diff
@@ -231,16 +223,25 @@ void EntityScriptServer::pushLogs() {
     }
 
     auto nodeList = DependencyManager::get<NodeList>();
+    QJsonDocument document;
+    document.setArray(buffer);
+    QString data(document.toJson());
     for (auto uuid : _logListeners) {
         auto node = nodeList->nodeWithUUID(uuid);
         if (node && node->getActiveSocket()) {
             auto packet = NLPacketList::create(PacketType::EntityServerScriptLog, QByteArray(), true, true);
-            packet->write(buffer.data(), buffer.size());
+            packet->write(data.toStdString().c_str(), strlen(data.toStdString().c_str()));
```

Comment:
How about:

```suggestion
            auto cstring = data.toStdString().c_str();
            packet->write(cstring, strlen(cstring));
```

Also, since NLPacketList is our own code, why don't we have an overload for QString instead of messing around with this?

Reply:
Thanks! This looks nicer :)
As for the overload it would be a nice thing to have in the future.

Reply:
Oh, turns out we do have it!

It's `writeString`. Takes a QString as argument.

Reply:
Sadly that one adds extra data and it breaks the current PR. I get an error message `[02/17 15:28:11] [WARNING] [default] EntityScriptServerLogClient::handleEntityServerScriptLogPacket: Cannot parse JSON:  "illegal value"  Contents:  "6\b"`.

Reply:
It's tested and fixed now :)